azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -221,7 +221,12 @@ internal open class CallCompositeActivity : AppCompatActivity() {
             }
         }
 
-        diContainerHolder.container.callCompositeActivityWeakReference = WeakReference(null)
+        try {
+            // This code was added to avoid memory leak
+            diContainerHolder.container.callCompositeActivityWeakReference = WeakReference(null)
+        } catch (e: Exception) {
+            // Ignore any exception from the user handler
```

Comment:
@iaulakh  could you please log an error.

Reply:
I can't see how this would throw, it's just setting a field.

I.e. it says (Ignore any exception from user handler) but there is no user handler here.


I would think this would be
```
  diContainerHolder.container.callCompositeActivityWeakReference.clear()
```

But really, that doesn't make sense, because it's a weak reference and it should auto-clear when there is no references to it. So if the WeakReference isn't clearing another reference is keeping it alive. (I.e. the Context is still leaking probably).

Reply:
can you share the stacktrace for the crash?